hash/vsmile_cart.xml

```diff
@@ -788,7 +788,7 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
 			<feature name="cart_type" value="lilac" />
 			<feature name="u1" value="U1" />
 			<feature name="u2" value="U2" />
-			<dataarea name="rom" size="8388608">
+			<dataarea name="rom" size="0x600000">
```

Comment:
As with the other one where you changed the size, have you checked the code to see whether it makes assumptions about the region size?